mayastor/src/subsys/nvmf/admin_cmd.rs

```diff
@@ -48,6 +55,21 @@ impl From<*mut c_void> for NvmfReq {
     }
 }
 
+/// Set the snapshot time in an spdk_nvme_cmd struct to the current time
+/// Returns seconds since Unix epoch
+pub fn set_snapshot_time(cmd: &mut spdk_nvme_cmd) -> u64 {
```

Comment:
Don't think this needs to be pub as the only consumer is in this file?

Reply:
It's also called from BdevHandle::create_snapshot.